inspire_dojson/hep/rules/bdFFT.py

```diff
@@ -24,60 +24,45 @@
 
 from __future__ import absolute_import, division, print_function
 
-from datetime import datetime
-
 from dojson import utils
 
-from inspire_utils.helpers import force_list, maybe_int
+from inspire_utils.helpers import force_list
 
 from ..model import hep, hep2marc
-from ...utils import absolute_url
+from ...utils import absolute_url, afs_url
 
 
-@hep.over('_fft', '^FFT[^%][^%]')
+@hep.over('documents', '^FFT[^%][^%]')
 @utils.for_each_value
-def _fft(self, key, value):
-    def _get_creation_datetime(value):
-        if value.get('s'):
-            dt = datetime.strptime(value['s'], '%Y-%m-%d %H:%M:%S')
-            return dt.isoformat()
-
-    is_context = value.get('f', '').endswith('context')
-    if is_context:
-        return
-
-    return {
-        'creation_datetime': _get_creation_datetime(value),
-        'description': value.get('d'),
-        'filename': value.get('n'),
-        'flags': force_list(value.get('o')),
-        'format': value.get('f'),
-        'path': value.get('a'),
-        'status': value.get('z'),
-        'type': value.get('t'),
-        'version': maybe_int(value.get('v')),
-    }
-
-
-@hep2marc.over('FFT', '^_fft$')
-@utils.for_each_value
-def _fft2marc(self, key, value):
-    def _get_s(value):
-        if value.get('creation_datetime'):
-            dt = datetime.strptime(value['creation_datetime'], '%Y-%m-%dT%H:%M:%S')
-            return dt.strftime('%Y-%m-%d %H:%M:%S')
-
-    return {
-        'a': value.get('path'),
-        'd': value.get('description'),
-        'f': value.get('format'),
-        'n': value.get('filename'),
-        'o': value.get('flags'),
-        's': _get_s(value),
-        't': value.get('type'),
-        'v': value.get('version'),
-        'z': value.get('status'),
-    }
+def documents(self, key, value):
+    """Populate the ``documents`` key.
+
+        Also populates the ``figures`` key through side effects.
+        """
+    def _is_hidden(value):
+        return 'HIDDEN' in value or None
```

Comment:
You'll probably want to `.upper()` this value to avoid false negatives (`'hidden'`). Don't trust that the data is correct!